app/pages/Watching/index.js

```diff
@@ -54,31 +129,101 @@ class Watching extends Component {
     }
   };
 
+  onReset = async () => {
+    const {isConfirmingReset} = this.state;
+    const {network, reset} = this.props;
+
+    if (isConfirmingReset) {
+      await reset(network);
+      this.setState({isConfirmingReset: false});
+    } else {
+      this.setState({isConfirmingReset: true});
+    }
+  };
+
   onClose = () => this.setState({isAddingName: false, name: ''});
 
+  getText = () => {
+    const {
+      isConfirmingReset,
+      isImporting,
+    } = this.state;
+
+    if (isConfirmingReset) {
+      return 'This will permanently erased your watchlist on your device!';
+    }
+
+    if (isImporting) {
+      return 'Importing watchlist...';
+    }
+
+    return 'Your watchlist does not transfer across devices!';
```

Comment:
Can we leave this blank?  I feel like if someone is installing a desktop application and thinks if they install it again on another computer we magically know what they did...actually gets annoyed at us...that's ok.